client/hotrod-client/src/main/java/org/infinispan/client/hotrod/MetadataValue.java

```diff
@@ -7,32 +7,6 @@
  * @author Tristan Tarrant
  * @since 5.2
  */
-public interface MetadataValue<V> extends VersionedValue<V> {
-
-   /**
-    *
-    * @return Time when entry was created. -1 for immortal entries.
-    */
-   long getCreated();
-
-   /**
-    *
-    * @return Lifespan of the entry in seconds. Negative values are interpreted as unlimited
-    *         lifespan.
-    */
-   int getLifespan();
-
-   /**
-    *
-    * @return Time when entry was last used. -1 for immortal entries.
-    */
-   long getLastUsed();
-
-   /**
-    *
-    * @return The maximum amount of time (in seconds) this key is allowed to be idle for before it
-    *         is considered as expired.
-    */
-   int getMaxIdle();
+public interface MetadataValue<V> extends VersionedValue<V>, Metadata {
```

Comment:
I've separated it out so that it can be used in the objects returned by the streaming API methods.